readthedocs/audit/templates/audit/list_logs.html

```diff
@@ -10,79 +10,105 @@
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">accepted an invitation</a>
+            {% endblocktrans %}
+          {% elif log.action == AuditLog.INVITATION_DECLINED %}
+            {% blocktrans trimmed with action=log.action user=log.user_html_display|safe %}
+              User {{ user }} <a href="?action={{ action }}">declined an invitation</a>
             {% endblocktrans %}
           {% endif %}
-        {% endif %}
-
-        {% trans "from" %}
-
-        <a href="?ip={{ log.ip }}" title="{{ log.browser }}">
-          <code>{{ log.ip }}</code>
-        </a>
-
-        {# If the authentication was on a doc domain. #}
-        {% if log.log_project_id %}
-          {% trans "on" %}
-          {% if log.project %}
-            <a href="{% url 'projects_detail' log.log_project_slug %}">
-              <code>{{ log.log_project_slug }}</code>
-            </a>
-          {% else %}
-            {# The original project has been deleted, don't link to it. #}
-            <code title="{{ log.resource }}">{{ log.log_project_slug }}</code>
-          {% endif %}
-        {% endif %}
+        </div>
 
         <span class="quiet right" title="{{ log.created|date:"DATETIME_FORMAT" }}">
           {% blocktrans trimmed with log.created|timesince as date %}
             {{ date }} ago
           {% endblocktrans %}
         </span>
+
+        {# Data about the log entry #}
+        <div>
+          {% if log.action == AuditLog.AUTHN or log.action == AuditLog.AUTHN_FAILURE or log.action == AuditLog.LOGOUT or log.action == AuditLog.PAGEVIEW or log.action == AuditLog.DOWNLOAD %}
+            {% if log.log_project_slug %}
+              {% trans 'project: ' %}
+              <code>{{ log.log_project_slug }}</code>
+              |
+            {% endif %}
+            {% if log.action == AuditLog.PAGEVIEW or log.action == AuditLog.DOWNLOAD %}
+              {% trans 'page: ' %}
+              <code title="{{ log.resource }}">{{ log.resource|truncatechars:30 }}</code>
+              |
+            {% endif %}
+          {% elif log.action == AuditLog.INVITATION_SENT or log.action == AuditLog.INVITATION_REVOKED or log.action == AuditLog.INVITATION_ACCEPTED or log.action == AuditLog.INVITATION_DECLINED %}
+            {% if log.data.to_user %}
+              {% trans 'to user: ' %}
+              <code>{{ log.data.to_user.username }}</code>
+            {% elif log.data.to_email %}
+              {% trans 'to email: ' %}
+              <code>{{ log.data.to_email }}</code>
+            {% endif %}
+
+            |
+
+            {% if log.data.object_type == "project" %}
+              {% trans 'project: ' %}
+              <code>{{ log.data.object.slug }}</code>
+
+              {% if log.data.object.organization %}
+                |
+                {% trans 'organization: ' %}
+                <code>{{ log.data.object.organization.slug }}</code>
```

Comment:
Same on slug here. Users never see project/organization slugs, we should show them what they normally see in the UI (project/org name, etc).